cli/tests/test_completion.rs

```diff
@@ -227,22 +301,28 @@ fn test_aliases_are_resolved() {
         .run_jj(["config", "set", "--repo", "aliases.b2", "['bookmark']"])
         .success();
 
-    let mut test_env = test_env;
-    test_env.add_env_var("COMPLETE", "fish");
-    let test_env = test_env;
-    let work_dir = test_env.work_dir("repo");
-
-    let output = work_dir.run_jj(["--", "jj", "b", "rename", "a"]);
-    insta::assert_snapshot!(output, @r"
-    aaa	(no description set)
-    [EOF]
-    ");
-
-    let output = work_dir.run_jj(["--", "jj", "b2", "rename", "a"]);
-    insta::assert_snapshot!(output, @r"
-    aaa	(no description set)
-    [EOF]
-    ");
+    insta::allow_duplicates! {
+        for output in [
+            work_dir.complete_at(shell, 3, &["jj", "b", "rename", "a"]),
+            work_dir.complete_at(shell, 3, &["jj", "b2", "rename", "a"]),
+        ] {
```

Comment:
nit: just flatten loop instead of using `allow_duplicates`? It's harder to debug if one of these samples failed.